src/react/ssr.html

```diff
@@ -6038,6 +6038,8 @@ <h2 class="post__comment-author">@username_301 wrote</h2>
 
   </section>
 
+  <script src="react.min.js"></script>
+  <script src="react-dom.min.js"></script>
   <script src="app.js" async></script>
```

Comment:
This one I'm not sure about. Can I make them all `async` but guarantee order?
If not I guess we'll have to bundle and add a minification step (and thus add Babel ES2015 transform).